docs/tools/terraform/reference/cookbook/kafka-karapace-recipe.rst

```diff
@@ -0,0 +1,70 @@
+Apache Kafka® with Karapace Schema Registry
```

Comment:
There's an heading warning, you might need to add `Karapace Schema Registry` in `.github/vale/styles/Aiven/capitalization_headings.yml`

Reply:
Thanks 

https://github.com/aiven/devportal/pull/960